### DISCUSSION TITLE
Stop/break the VRP program if no solution is found

### DISCUSSION THREAD

**Discussion by Westlife1002** [google/or-tools#2550] [Routing (and legacy CP) questions]
I have an GUI for the VRP model, developed based on tkinter. 
When clicking the "run" button and if there is "no solution", it gets stuck and no way to exit and to print ("no solution"). I guess it's the VRP programming still running. Is there any way to stop the VRP part of the program if no solution is found, and also print ("no solution")

my lastest codes is here:
https://gist.github.com/Westlife1002/03f5f2a1b03841a81221623bce63951e

**Comment by Mizux** [comment]
For the record, seems a followup of the #2539 issue
